nslsii/__init__.py

```diff
@@ -212,15 +215,14 @@ def configure_base(
             RE,
             beamline_name=broker_name,
             bootstrap_servers="cmb01:9092,cmb02:9092,cmb03:9092",
-            producer_config={
-                "enable.idempotence": True,
-                "linger.ms": 0
-            }
+            producer_config={"enable.idempotence": True, "linger.ms": 0},
         )
 
-    # always configure %xmode minimal
-    # so short tracebacks are printed to the console
-    get_ipython().magic("xmode minimal")
+    # always
```

Comment:
Was it left after moving comments around?

Reply:
oops.